### DISCUSSION TITLE
Filter /snap Partition on check_drivesize

### DISCUSSION THREAD

**Discussion by Droning1355** [ConSol-Monitoring/snclient#160] [Q&A]
Hello everybody,

i need a little bit help for filtering out the snap partitions on Linux systems.
i have tried to filter the partition with name filters.

i use the nrp command:
$USER1$/check_nrpe -H $HOSTADDRESS$ -p xxxx -A /etc/ssl/snclient/cacert.pem -C /etc/ssl/snclient/server.pem -K /etc/ssl/snclient/server.key -c $ARG1$

and the check_drivesize:
![image](https://github.com/user-attachments/assets/753a4c08-0b74-438f-a4c5-410a98efd9ca)

the parameter -a mouted was my last try. before this my filter was like this:
-a "filter= name not in /snap" 
I have tried some more but sadly i forget to document them.

thanks for the replies

**Comment by sni** [comment]
i assume those mounts start with /snap, so i guess you need something like:
"filter= name unlike ^/snap/"

If that doesn't help, how do the names look like?
